log/log.go

```diff
@@ -0,0 +1,88 @@
+package log
+
+import (
+	"context"
+	"log"
+	"os"
+	"strings"
+	"testing"
+
+	"github.com/docker/docker/client"
+)
+
+// Logger defines the Logger interface
+type Logger interface {
+	Printf(format string, v ...any)
+	Print(v ...any)
```

Comment:
question: does this need to include `Print`?

Reply:
Strictly no, it was sort of replacing some `Println` uses in `wait` but that can also be achieved with `Printf`. Happy to remove so the interface remains the same that it was, so it doesn't break any existing user implementations.